backend/tests/unit/api/Controllers/Tools/ImportControllerTest.cs

```diff
@@ -196,7 +197,7 @@ public void ImportProjects_Success()
             };
 
             // Act
-            var result = controller.ImportProjects(projects, false);
+            var result = controller.ImportProjects(projects, false, DateTime.UtcNow);
```

Comment:
just curious, why UTC time?

Reply:
We always store dates in the DB with a single standard time.  The convention is UTC.  If you store different timezone values in the DB you then also need to store what timezone you stored the date.  Which historically was problematic and unreliable.